lib/src/posthog.dart

```diff
@@ -30,9 +28,9 @@ class Posthog {
   }
 
   Future<void> capture({
-    @required String eventName,
-    Map<String, dynamic> properties,
-    Map<String, dynamic> options,
+    required String eventName,
+    required Map<String, dynamic> properties,
```

Comment:
this seems to have diverged from lib/src/posthog_method_channel.dart where properties are correctly assigned as optional

Reply:
You're right, I'm going to make it nullable instead of required